WEBSITE-6: Website updated

CBibop12 opened the pull request.

CBibop12 committed:
WEBSITE-6: Website updated

CBibop12 committed:
website polishing

bbrands02 reviewed (changes requested):
<img width="1064" height="1030" alt="image" src="https://github.com/user-attachments/assets/e86f31fe-5714-41a6-ae28-a0fcef7c2bc7" />
<img width="1154" height="757" alt="image" src="https://github.com/user-attachments/assets/0d66b817-cb64-4048-b73e-17ea49431bbc" />
<img width="485" height="573" alt="image" src="https://github.com/user-attachments/assets/fcb42d9c-efd6-4293-8b9a-f9dc196f6fd3" />

CBibop12 committed:
last upd

rubenvdlinde committed:
Remove DashKube training card

Per review on PR #3: DashKube is a deprecated project, the standalone
training card on /trainingen has been removed.

rubenvdlinde committed:
Replace stray dashkube.png with openCatalogi.png on Softwarecatalogus

The VNG Softwarecatalogus project card was showing /img/dashkube.png
as its mainImage, which prompted the reviewer to think DashKube was
listed as a project. Description already states the catalogue 'is
gebaseerd op OpenCatalogi', so /img/beheerCarousel/openCatalogi.png
is the thematic match.

rubenvdlinde commented:
@bbrands02 — review afgewerkt, samengevat per punt:

| # | Review-punt | Status | Commit |
|---|---|---|---|
| 1 | Typo `WaardPapieren` → `WaardePapieren` | Al opgelost door @CBibop12 | [`2511734`](../commit/2511734) |
| 2 | "Basis Common Ground" knop linkte hardcoded naar de live site | Al opgelost: `TrainingCard` gebruikt nu `<Link to={link}>` i.p.v. `<a href target="_blank">`, en `link: '/training-basis-common-ground'` (idem voor de andere twee trainingen) is toegevoegd. | [`2511734`](../commit/2511734) |
| 3 | "NEEM CONTACT OP" op `/over-ons` werkte niet | Al opgelost: `<Link to="/contact">` (was `/over-ons`) | [`2511734`](../commit/2511734) |
| 4 | DashKube training card op `/trainingen` weghalen | Nu verwijderd | [`78b6c1d`](../commit/78b6c1d) |
| 5 | DashKube op `/projecten` weghalen | Toelichting: er stond geen DashKube-projectblok in deze branch — alleen `/img/dashkube.png` als verkeerde `mainImage` op de **VNG Softwarecatalogus** kaart. Vervangen door `/img/beheerCarousel/openCatalogi.png` (de beschrijving zelf zegt al *"is gebaseerd op OpenCatalogi"*). | [`fb799fe`](../commit/fb799fe) |

Punten 1–3 had Alex op 31 oktober al gefixt in `2511734 "last upd"`, maar de review was niet als resolved gemarkeerd; daardoor stond de PR nog op Changes Requested.

Re-review graag — als de image-vervanging op punt 5 niet de juiste oplossing is, hoor ik graag wat het wel moet zijn (er was geen VNG-Softwarecatalogus-specifiek plaatje in `static/img/`).

rubenvdlinde committed:
Remove trainingen pages, nav link, and homepage/common-ground references

Trainings are no longer offered as a service. Removes the 4 trainingen
pages (trainingen, training-basis-common-ground, training-hoe-te-beginnen,
training-open-source-overheid), the TrainingCard component, the nav link
in docusaurus.config.js, the Trainingen card on the homepage, the
Trainingen section on /common-ground, and all related images under
static/img/training, static/img/trainingDetails, and trainingenHero.png.

Unused .trainingCard* CSS classes in index.module.css are left in place.

rubenvdlinde committed:
Add team portraits for Mark, Remco, Remko and Thijn

rubenvdlinde committed:
Update team on /over-ons: remove Matthias, add Remco, Mark, Remko, Thijn

Matthias is no longer at Conduction; his card is removed.

Adds four new team members in the team section in directie -> sales ->
solutions -> development order:
  Row 1: Ruben, Marleen, Remco (sales en operations)
  Row 2: Mark (solutions), Robert, Wilco
  Row 3: Barry, Remko (frontend), Thijn (frontend)

Adds a GitHub link to every team card alongside existing LinkedIn (where
known) and email links. New cards omit LinkedIn pending URLs; bios are
placeholder copy that should be reviewed by each person.

rubenvdlinde commented:
Vervolg op de eerdere commit-batch — drie aanvullende wijzigingen op vraag van @rubenvdlinde, **niet uit Barry's review** maar wel logisch in dezelfde PR:

| Wijziging | Commit |
|---|---|
| Trainingen volledig verwijderd: 4 pages (`trainingen.js`, `training-basis-common-ground.js`, `training-hoe-te-beginnen.js`, `training-open-source-overheid.js`), `TrainingCard` component, navlink, homepage-card, common-ground-card, `static/img/training/`, `static/img/trainingDetails/`, `trainingenHero.png`. (Ongebruikte `.trainingCard*` CSS-classes in `index.module.css` blijven staan — buiten scope.) | [`aedef62`](../commit/aedef62) |
| Portretten voor 4 nieuwe team-leden toegevoegd (Mark, Remco, Remko, Thijn) van Slack-avatars; `matthias.png` verwijderd. | [`4f8337d`](../commit/4f8337d) |
| `/over-ons` team-sectie herzien: Matthias-kaart verwijderd, 4 kaarten toegevoegd, alles in volgorde **directie → sales → solutions → development**, en op elke kaart een **GitHub-link** toegevoegd naast LinkedIn (waar bekend) en e-mail. | [`efbb046`](../commit/efbb046) |

**Aandachtspunten voor merge:**
- **Bios zijn placeholder** voor de 4 nieuwe mensen — gebaseerd op rol, niet op input van henzelf. Elke nieuwe collega zou de eigen tekst even moeten reviewen.
- **LinkedIn-URL's** ontbreken voor de 4 nieuwe kaarten (ik heb ze niet gegokt om dode/foute links te voorkomen). Aanvullen kan in een follow-up.
- **GitHub-mappings** die ik heb gebruikt (klopt dit?):
  - Ruben → [`rubenvdlinde`](https://github.com/rubenvdlinde) · Marleen → [`Marleen73`](https://github.com/Marleen73)
  - Remco → [`Rem-Dam`](https://github.com/Rem-Dam) · Mark → [`MWest2020`](https://github.com/MWest2020)
  - Robert → [`rjzondervan`](https://github.com/rjzondervan) · Wilco → [`WilcoLouwerse`](https://github.com/WilcoLouwerse) · Barry → [`bbrands02`](https://github.com/bbrands02)
  - Remko → [`remko48`](https://github.com/remko48) · Thijn → [`SudoThijn`](https://github.com/SudoThijn)
- **Mark's avatar** kwam als JPEG van Slack-CDN; via PIL geconverteerd naar echte PNG zodat alle team-images consistent zijn.

rubenvdlinde commented:
GitHub-mappings geverifieerd via recente commit-activiteit op ConductionNL repos — alle 9 kloppen:

| Persoon | GitHub | Laatste ConductionNL commit |
|---|---|---|
| Ruben | [`rubenvdlinde`](https://github.com/rubenvdlinde) | 2026-04-28 |
| Marleen | [`Marleen73`](https://github.com/Marleen73) | 2020-11-15 (rol is non-coding) |
| Remco | [`Rem-Dam`](https://github.com/Rem-Dam) | 2026-02-16 |
| Mark | [`MWest2020`](https://github.com/MWest2020) | 2026-04-20 |
| Robert | [`rjzondervan`](https://github.com/rjzondervan) | 2026-04-23 |
| Wilco | [`WilcoLouwerse`](https://github.com/WilcoLouwerse) | 2026-04-09 |
| Barry | [`bbrands02`](https://github.com/bbrands02) | 2026-04-15 |
| Remko | [`remko48`](https://github.com/remko48) | 2026-04-23 |
| Thijn | [`SudoThijn`](https://github.com/SudoThijn) | 2026-02-11 |

Geen verdere code-wijzigingen nodig.
